### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by Buedenbender** [rempsyc/lavaanExtra#6] [Show and tell]
Just used the package again for an analysis. So convenient, just wanted to thank you 👍 

```
lavaan_ind(fsem_complete, nice_table = TRUE)
nice_lavaanPlot(fsem_complete)
```

![grafik](https://user-images.githubusercontent.com/38650857/196442821-87002a7d-82bd-48c8-b6b7-2e2119916140.png)

**Comment by rempsyc** [comment]
Try updating to the latest version and do this table again see if you see any difference :P

**Comment by rempsyc** [comment]
Would be happy to hear any feedback if you think there are other ways to improve the default output. I'm actually not sure what a typical indirect effects table should look like for publications!

**Comment by Buedenbender** [comment]
Oh awesome, looks like this:

![grafik](https://user-images.githubusercontent.com/38650857/196457430-14d8c62d-24f7-4daf-9fac-5ba031934b75.png)


But this is my fault for naming my indirect effects like this:

        # ...
        # Age Effect
        id_age_negative := p2 * p8
        id_age_positive := p1 * p7
        # ...

But this is fixed fast. I really like the changes.
Actually I realized I would not know either, because most of the times I see path analyses / SEM it is not accompanied by
a table of the effects but they are usually displayed in a figure and/or in the text (e.g., Nicolai et al., 2016). Most of the times, I did not see an explicit table on the effect sizes (I think this is due to the fact, that you probably would not want to repeat the same information to many times: in a figure, table and the text). 

Nicolai, J., Buchholz, A., Seefried, N., Reuter, K., Härter, M., Eich, W., & Bieber, C. (2016). When do cancer patients regret their treatment decision? A path analysis of the influence of clinicians’ communication styles and the match of decision-making styles on decision regret. Patient Education and Counseling. https://doi.org/10.1016/j.pec.2015.11.019
